<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_font">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/wall2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_fecha">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>454</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_senal_celular">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>457</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/no3G.Jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_conex_gps">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>457</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/noGPS.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_state_simcard">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>457</y>
     <width>34</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/noSIM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>456</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/noRed.Jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>457</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/noSocket.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>457</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_ser">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>452</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NUMERO DE SERIE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_ver">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>462</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>VERSION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ser_pc">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>410</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SERIE PC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_unidad">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>UNIDAD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_off">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/apagar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_version_software">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EL.v1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_socket">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Socket</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_datos_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
background-color: rgb(206, 206, 206);</string>
   </property>
   <property name="text">
    <string>No apague la boletera, datos a√∫n no enviados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_datos_cantidad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
background-color: rgb(206, 206, 206);</string>
   </property>
   <property name="text">
    <string>faltan 3 datos por enviar.</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="Brillo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:horizontal {
	border: none;
    background: rgb(213, 213, 213);
    width: 14px;
    margin: 5px 0 5px 0;
	border-radius: 0px;
 }

QScrollBar::handle:horizontal {	
	background-color: rgb(28, 83, 165);
	min-height: 300px;
	border-radius: 7px;
}

QScrollBar::handle:horizontal:hover{	
	background-color: rgb(246, 130, 31);
}

QScrollBar::handle:horizontal:pressed {	
	background-color: rgb(246, 130, 31);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="reset_nfc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/circular.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
